.travis.yml

```diff
@@ -4,7 +4,7 @@ compiler:
   - gcc
 notifications:
   email:
-    on_success: change
+    on_success: always
     on_failure: always
     recipients:
       - ros-contributions@amazon.com
```

Comment:
I asked the question to the team before: do we really want these always on-success emails sent to "ros-contributions@amazon.com"?

Reply:
I think we said we'd try different settings and see if Travis supports that - I don't remember if we actually tried it though? e.g.:

```
  email:
    on_success: change
    on_failure: always
    recipients:
      - ros-contributions@amazon.com
  email:
    on_success: always
    on_failure: always
    recipients:
      - <the ticketing email>
```

If it doesn't work, and folks don't like the idea of an inbox filter, we could set up another alias like robomaker-platform-ops@ / robomaker-platform-builds@ or something.

Reply:
I already have it filtered. With the other infinite automated emails we have set up, I can't imagine anyone not having decent filters set up.

Reply:
One disadvantage of not having it go to our inboxes is that it might make debugging these emails more difficult in the future. I was only able to figure out what the problem is here because I was able to look through my own inbox.

Reply:
Okay, let's do it then. (I don't have confidence that @AAlon's example, will work even though it's reasonable.)